### DISCUSSION TITLE
Feature: Use floors as Magic Areas

### DISCUSSION THREAD

**Discussion by Patrick1610** [jseidl/magic-areas#327] [Ideas / Feature Requests]
Currently, Home Assistant allows you to specify floors in which areas are placed. The floors don't show up as areas and it is not possible to select them when creating a Magic Area. I would really like to be able to do so, so that I can use automations based on the state of an entire floor. 

Would it be possible to add that to Magic Areas?

![image](https://github.com/jseidl/hass-magic_areas/assets/6585375/0b868744-bc69-4781-a38d-1eddbf037a34)

**Comment by tbrasser** [comment]
Probably floors should be meta areas like interior/exterior/global

**Reply by Patrick1610** [reply]
That would also be just fine! I will still be able to use them in automations, which is my goal =)

**Comment by jseidl** [comment]
Yes making meta areas for floors is probably what I'm going to work on next. Just have been busy getting linters and tests going on to make everything more reliable

**Comment by jseidl** [comment]
This is coming up soon! PR https://github.com/jseidl/hass-magic_areas/pull/342
